src/components/RelatedMeetups.js

```diff
@@ -0,0 +1,112 @@
+import React from 'react';
+import { useStaticQuery, graphql } from 'gatsby';
+import Img from "gatsby-image"
+
+const RelatedMeetups = () => {
+	const data = useStaticQuery(graphql`
```

Comment:
Look into querying allImageSharp and see if you can add a relativePah to that and get all that you are looking for in 1 query.